### PR TITLE
Fix broken PKICertImport test

### PR DESCRIPTION
**Pull request by fmarco76**
The PKICertImport test was failing because the certificate provided in the repository were encoded with obsolete cipher not supported by current version of OpenSSL unless the `-legacy` option is provided.

To solve the issue the certificates have been removed and e new method to generate the certificate has been integrated. This allows an easier update of the algorithms used in the certificate.

All the step to generate the certificate are performed with `pki` command. Some test could also be replaced to use `pki` but this could be in a different PR.

### CODE REVIEW

**Comment by ckelleyRH** [comment]
Guess we need to wait until the CI is restored to see how the test runs :-)

**Comment by fmarco76** [comment]
Yes, I tested locally only the failing test but I would perform all the other test before to merge so it is better to wait. I do not see an option to run all the test again, is this possible or they should be done manually each one?

**Comment by ckelleyRH** [comment]
Unfortunately not, but if you prefer typing to clicking, there is a github CLI where you can type `gh run rerun` to choose a workflow to re-run.

**Comment by edewata** [comment]
Alternatively you can just amend the commit (with nothing changed) then force push the branch.

File: base/tools/src/test/resources/certs/ca_signing.conf
Comment (edewata) [inline base/tools/src/test/resources/certs/ca_signing.conf:1]: I'd suggest we use [example.com](https://en.wikipedia.org/wiki/Example.com) for all examples. Could you align the `=` too for consistency? Thanks.

Comment (fmarco76) [reply base/tools/src/test/resources/certs/ca_signing.conf]: Updated!

File: base/tools/src/test/shell/test_PKICertImport.bash
Comment (edewata) [inline base/tools/src/test/shell/test_PKICertImport.bash:67]: Could we use example.com here as well?

File: base/tools/src/test/shell/test_PKICertImport.bash
Comment (edewata) [inline base/tools/src/test/shell/test_PKICertImport.bash:556]: Could you adjust the indentation too?

**Review by edewata — APPROVED** [review]
Nice! I have a few more comments, but I think it LGTM. Feel free to update and merge. Thanks!

Note: The remaining test failures are intermittent, you can ignore them.

File: base/tools/src/test/shell/test_PKICertImport.bash
Comment (fmarco76) [reply base/tools/src/test/shell/test_PKICertImport.bash]: Yes, it is better to uniform. Updated!

File: base/tools/src/test/shell/test_PKICertImport.bash
Comment (fmarco76) [reply base/tools/src/test/shell/test_PKICertImport.bash:556]: This is strange. If I open with the my editors (`vim` and `emacs`) is correct, spaces all lines. If I go to file changed is not correct but if I click on view the file is correct.

I think it is a visualisation  problem!

Comment (fmarco76) [reply base/tools/src/test/shell/test_PKICertImport.bash]: At the end I change a different line and it is updated.

**Review by ckelleyRH — APPROVED** [review]

**Comment by fmarco76** [comment]
Thanks @ckelleyRH @edewata

**Pull request merged**
